instrumentation/jsonrpc4j-1.6/javaagent/src/test/java/io/opentelemetry/javaagent/instrumentation/jsonrpc4j/v1_6/AgentJsonRpcTest.java

```diff
@@ -0,0 +1,28 @@
+/*
+ * Copyright The OpenTelemetry Authors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package io.opentelemetry.javaagent.instrumentation.jsonrpc4j.v1_6;
+
+import com.googlecode.jsonrpc4j.JsonRpcBasicServer;
+import io.opentelemetry.instrumentation.jsonrpc4j.v1_6.AbstractJsonRpcTest;
+import io.opentelemetry.instrumentation.testing.junit.AgentInstrumentationExtension;
+import io.opentelemetry.instrumentation.testing.junit.InstrumentationExtension;
+import org.junit.jupiter.api.extension.RegisterExtension;
+
+public class AgentJsonRpcTest extends AbstractJsonRpcTest {
```

Comment:
```suggestion
class AgentJsonRpcTest extends AbstractJsonRpcTest {
```
Please pay attention to other similar places, we often keep the visibility of fields, classes, and methods as minimal as possible.